scripts/manifest

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
What are we gaining with this change?

Reply:
Consistency across all shell scripts in the scripts/ directory:
```
brandond@dev01:/var/lib/repos/go/src/github.com/rancher/k3s$ head -n 1 scripts/* | grep -F '#!' | sort | uniq -c
     29 #!/bin/bash
      3 #!/bin/sh
      1 #!/usr/bin/env bash
```